### ISSUE TITLE
Your credentials class does not support session injection.

### ISSUE THREAD

**Issue by sr9yar** [Azure-Samples/batch-python-quickstart#11]
### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [x] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> 

Run the program as-is

### Any log messages given by the failure
>
".Your credentials class does not support session injection. Performance will not be at the maximum."

### Expected/desired behavior
>
Absence of this warning 

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.1 LTS"

### Versions
>

python --version
Python 3.6.7

### Mention any other details that might be useful

When I try to run the code, I'm getting this warning output. For now I was unable to find any explanation regarding creds usage. The files are created in the storage container ok. 
Is this an expected behavior?
Example of the output I get: 

```
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.
.Your credentials class does not support session injection. Performance will not be at the maximum.

  Success! All tasks reached the 'Completed' state within the specified timeout period.
Printing task output...
Your credentials class does not support session injection. Performance will not be at the maximum.
Your credentials class does not support session injection. Performance will not be at the maximum.
Task: Task0
Node: tvm-1347421722_1-20181203t153003z
Your credentials class does not support session injection. Performance will not be at the maximum.
Standard output:
Traceback (most recent call last):
  File "python_quickstart_client.py", line 374, in <module>
    print_task_output(batch_client, config._JOB_ID)
  File "python_quickstart_client.py", line 287, in print_task_output
    print(file_text)
UnicodeEncodeError: 'ascii' codec can't encode character '\ufeff' in position 0: ordinal not in range(128)


```

**Comment by laurenhughes** [comment]
Thanks bringing this up, @sr9yar. I submitted a fix (#12) to update the Batch SDK for Python version that should fix the issue.

**Comment by laurenhughes** [comment]
I'm going to go ahead and close the issue. If you have any other questions, please feel free to comment here!
